e2e/filters.spec.ts

```diff
@@ -85,4 +84,38 @@ describe("Filters", () => {
     }
     await page.getByRole("button", { name: "2014", exact: true }).click();
   });
+  test("Show legend titles, should toggle legend titles visibility", async ({
+    page,
+    actions,
+    selectors,
+  }) => {
+    test.slow();
+
+    await page.goto(
+      "/en/create/new?cube=https://energy.ld.admin.ch/sfoe/bfe_ogd84_einmalverguetung_fuer_photovoltaikanlagen/5&dataSource=Prod"
+    );
+    await selectors.chart.loaded();
+
+    await actions.editor.selectActiveField("Segmentation");
+
+    await selectors.edition.drawerLoaded();
+
+    await (await selectors.panels.drawer().within().findByText("None")).click();
+
+    await actions.mui.selectOption("Kanton");
+
+    const legend = page.locator('[data-testId="legend-container"]');
```

Comment:
@noahonyejese same thing as previously, I think it should be only legend title that is shown / hidden, not the whole container 👍